### DISCUSSION TITLE
Welcome to pymc-experimental Discussions!

### DISCUSSION THREAD

**Discussion by fonnesbeck** [pymc-devs/pymc-extras#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ccaprani** [comment]
Thanks Chris. Briefly, my specific interests are in modelling extreme values, which hasn't typically been strongly bayesian; but more broadly keen to see *the logic of science* be more widely adopted in engineering where it is hugely needed.

I think this repo could really work for pymc - allowing the main package to be more stable and also providing a lower bar to contributions with less stress for maintainers. Getting clear agreed answers to the questions posed in the ReadMe will be vital - so I'll add a discussion post on each of these to collate thoughts.
